apps/src/tutorialExplorer/util.jsx

```diff
@@ -33,4 +33,22 @@ function getTagString(prefix, tagString) {
   return tagString.split(',').map(tag => tagToString[`${prefix}_${tag}`]).join(', ');
 }
 
-export default getTagString;
+/**
+ * Given a tutorial item, return the string to render for its details.
+ * @param {object} item - A tutorial item.
+ * @return {string} - The detail string, e.g. "Grade 4 | C++ | Web" or "Grade 4 | C++".
+ */
+
+function getTutorialComboString(item) {
```

Comment:
If you wanted to get a little fancy you could do something like:
```
function getTutorialComboString({string_detail_grades, string_detail_programming_languages, string_detail_platforms}) {
```
I don't have any strong preference for either (might prefer your approach just bc names are so long)